tests/module/network/src/main.c

```diff
@@ -33,6 +34,7 @@ ZBUS_MSG_SUBSCRIBER_DEFINE(test_subscriber);
 ZBUS_CHAN_ADD_OBS(PAYLOAD_CHAN, test_subscriber, 0);
 
 #define FAKE_TIME_MS 1716552398505
+#define FAKE_RSRP_IDX  -16
```

Comment:
```suggestion
#define FAKE_RSRP_IDX  -96
```

Use a realistic (and possible) value.

Reply:
This is the faked RSRP 'index' that we should get from the modem. This gets converted to -156 dBm which is a valid value.

Reply:
Ah, ok. -156 dBm is not valid according to the docs, but OK. In that case we can extend the range in the CDDL.